src/tests/end-to-end/common/element-identifiers/common-selectors.ts

```diff
@@ -11,3 +11,10 @@ export const CommonSelectors = {
 export const GuidanceContentSelectors = {
     mainContentContainer: '.content-container',
 };
+
+export const fastPassSelectors = {
```

Comment:
I think if these are specific to fast pass, it might be better to put them in a fastpass-specific selectors file, rather than common-selectors.ts

Reply:
Done.